push-subscription-management/index.js

```diff
@@ -1,5 +1,8 @@
+// [Working example](/serviceworker-cookbook/push-subscription-management/).
+
 var subscriptionButton = document.getElementById('subscriptionButton');
 
+// Provide a Promise with subscription to not repeat too often
```

Comment:
As `subscription` object is needed in few places let's create a method which returns a promise.